py-scripts/real_application_tests/real_browser/lf_interop_real_browser_test.py

```diff
@@ -1042,304 +1403,309 @@ def updating_webui_runningjson(self, obj):
             json.dump(data, file, indent=4)
 
     def create_report(self,):
-        if self.dowebgui:
```

Comment:
If moving this to a try catch block, first please move it into a separate commit. Additionally please also simplify the try catch to only include the relevant section which may throw an exception.